CONTRIBUTING.md

```diff
@@ -41,27 +41,27 @@ to know that you have a clean state.
   ```sh
   # if your patch is browser specific
   # (e.g. upload files)
-  grunt test:desktop
+  npm run test:desktop
 
   # if your patch is mobile specific
   # (e.g. flick or swipe tests)
-  grunt test:ios test:android
+  npm run test:mobile
```

Comment:
This should be `npm run test:ios` or `npm run test:android` (see [here](https://github.com/webdriverio/webdriverio/blob/master/package.json#L46))